spec/features/users/user_can_login_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'rails_helper'
+
+
+describe "user can loggin" do
+  it "when nthey login" do
+    user = User.create(first_name: "Charlotte",
+                       last_name: "Moore",
+                       username: "Cj",
+                       email: "email@email.com",
+                       password: "password")
+    visit login_path
+    fill_in "session[username]", with: user.username
+    fill_in "session[password]", with: user.password
+
+    save_and_open_page
+    within(".login-submit") do
```

Comment:
I like the css selector on the test, we probably need to make this standard for our tests